ssl/statem/statem_srvr.c

```diff
@@ -1618,17 +1618,6 @@ MSG_PROCESS_RETURN tls_process_client_hello(SSL_CONNECTION *s, PACKET *pkt)
                 SSLfatal(s, SSL_AD_INTERNAL_ERROR, ERR_R_INTERNAL_ERROR);
                 goto err;
             }
-            /*
```

Comment:
Should this be conditional on DTLS-1.3 being enabled?

Reply:
What do you mean by "being enabled"?

I cannot check for connection versions at this point because the server has not yet called `ssl_choose_server_version()`.

But maybe I can move the check to a place after the extensions has been read and then check for a cookie then?

Reply:
I meant DTLS-1.3 not being enabled on the SSL_CONNECTION object (i.e. what ssl_get_min_max_version() returns).

Reply:
Please check my latest update and let me know your thoughts.